lib/soap/request/options.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Soap.Request.Options do
```

Comment:
We do not need a structure for the request body and headers.
These parameters are transmitted once and no longer used

Reply:
Done.